lib/generators/manageiq/plugin/plugin_generator.rb

```diff
@@ -81,6 +81,14 @@ def plugin_description
       @plugin_description ||= "#{file_name.titleize} plugin for #{Vmdb::Appliance.PRODUCT_NAME}."
     end
 
+    def template_unless_exists(destination)
+      if File.exist?(File.join(destination_root, destination))
+        say_status :exist, destination, :blue
+      else
+        template(destination)
```

Comment:
`template` will check for existence, but if there are any changes, it will flag it as changed, mark it red, and ask for override.  Since the file is empty by default, _any_ changes are local we don't ever want to override, so this short-circuits it with an existence check.

Reply:
@Fryguy So, there is a little more to this than the link I am providing, but I think you can avoid the custom method here and just pass the proper options to `template` (method is originally from `Thor`) so that it will skip by default if it exists:

https://github.com/erikhuda/thor/blob/34df888d721ecaa8cf0cea97d51dc6c388002742/lib/thor/actions/create_file.rb#L72-L79

Reply:
I saw that, but I can't figure out how to get the behavior I want.  Let me try passing :skip and see if that works.

Reply:
So I think for this, you would just pass `:skip => true` to the end of the `template` method instead of replacing it with this method.

Reply:
Oh nice, that worked!  I must have been passing the option wrong the first time though.